docs/usage/java.md

```diff
@@ -36,7 +78,12 @@ Renovate can update dependency versions found in Maven `pom.xml` files.
 
 Renovate will search repositories for all `pom.xml` files and processes them independently.
 
-### Custom registry support, and authentication
+## Custom registry support, and authentication
```

Comment:
```suggestion
### Custom registry support, and authentication
```

I think this section is not `Maven File Support` specific, and so should be it's own `h3` section, instead of being in the `Maven File Support` category.

Reply:
I might be wrong here, but the change you suggest seems to achieve the `Custom registry support, and authentication` being a part of `Maven`.
Thus, achieving the opposite of what you've written.

To me, the part is neither specific to the managers `gradle` nor `maven` as it applies to the datasource `maven` instead, which is used by both managers. Might be in the wrong here though.

Reply:
> I might be wrong here, but the change you suggest seems to achieve the `Custom registry support, and authentication` being a part of `Maven`.
> Thus, achieving the opposite of what you've written.

You're right! I got things mixed up... 🙈 

Ignore my previous suggestion! 😉

Reply:
Alright. I'll mark this as resolved than. Thanks for the quick review & response. 👍🏻🙂

Reply:
If this belongs to maven datasource, then we should move it there in a seperate docs PR.